src/App.tsx

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
+interface State {
+  selectedGood: string | null,
```

Comment:
string that is blank will return false when converted to boolean, so you don't need to check for null here
```suggestion
  selectedGood: string,
```